### ISSUE TITLE
Support to skip task

### ISSUE THREAD

**Issue by eduardomourar** [OlafConijn/AwsOrganizationFormation#68]
There are certain situations where you would like to keep some resources locked to a particular version. I believe we can achieve that by having a property called `skip` or `locked` within a task that will simply keep that resource in the current saved state.

This is very important for production environments where you want to prevent unwanted updates. We should use [stack policies](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/protect-stack-resources.html) from CloudFormation services itself to prevent changes from org-formation or from anyone with access to the stacks in the account.

**Comment by OlafConijn** [comment]
added a `Skip` attribute to the tasks.

I think it makes sense to have the `StackPolicy` be part of the update-stack. Will have to look into whether, when running a CDK or SLS task, it is also possible to retrieve the stack name(s) and targets in order to apply a StackPolicy to the stacks.

Another method of preventing updates to stacks (apart from deletion protection - which is already supported) could be by a lock-stacks task type.

Having a lock-stacks with a StackName and binding would allow the task to create a SCP that prevents updates to the stack in any of the targets enumerable from binding.

which would have your preference?

**Comment by eduardomourar** [comment]
I do like the idea of lock-stacks if we can use it for stacks that have been deployed by org-formation or not (I believe that is the idea, right?).

**Comment by OlafConijn** [comment]
yes, that would be the idea.

What makes me wonder though... what would be the benefit of having a lock-stacks task as opposed to a SCP in the organization file that effectively does the same?

i mean, i would be able to add an SCP like the following regardless of any task i have:
```
  RestrictUnusedRegionsSCP:
    Type: OC::ORG::ServiceControlPolicy
    Properties:
      PolicyName: RestrictUpdatesOnProtectedStacks
      PolicyDocument:
        Version: '2012-10-17'
        Statement:
          - Sid: RestrictCfnUpdates
            Effect: Deny
            Action: 'cloudformation:Update*'
            Resource: 'arn:aws:cloudformation:*:*:stack/protect-me*/*'
```

Meanwhile implementing the StackPolicy - more than anything because it is an integral part of CloudFormation.

**Comment by OlafConijn** [comment]
released as part of v0.9.5
